.github/workflows/build.yml

```diff
@@ -3,12 +3,12 @@ name: Continuous integration
 on:
   push:
     paths-ignore:
-      - "CHANGELOG.adoc"
```

Comment:
Is there a reason to replace double quotes with single quotes?
Double quotes are used in other workflows.

Reply:
no there is not, the script did that automatically

Reply:
ok